src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md

```diff
@@ -3,369 +3,191 @@ path: "/docs/writing-own-tests/setting-up-galasa-project"
-            <scope>compile</scope>
-        </dependency>
-    </dependencies>
-</project>
-```
 
-- `<packaging>` is set to `galasa-obr` which causes the Galasa Maven plugin to build this project.
-- Any custom Manager and test projects need to be included so that an OBR and test catalog is built after examination of the contents of these projects.
+## About the test projects
 
-</details>
+Within each of the Galasa test projects  - `payee` and `account` - you can see the following files and folders:
 
-## Importing the prepared directory into Eclipse
+- A pom.xml file (for use by the Maven build tool)
 
-Launch Eclipse and choose _File > Import..._
+- A `src` tree holding source code
 
-In the _Select_ dialog, expand _Maven_, choose _Existing Maven Projects_ and click _Next_.
+- Two java files in each `feature` test project
+
+- A text resource file in each `feature` test project, which is read and used by the tests at run-time.
+
+
+## About the tests
+
+The _TestAccount.java_ and the _TestPayee.java_ source files show how a Core Manager can be injected into your test class before any test methods being used. 
+
+The _TestAccountExtended.java_ and the _TestPayeeExtended.java_ source files show:
+
+- How to obtain the `run-id` identifier by which an instance of running the test can be known. This can be useful for naming artifacts, logging or otherwise.
+
+- How to read a text file resource which is embedded within the test OSGi bundle. This can be useful for getting data files for use in the testing of the application.
+
+- How logging can be performed to help debug issues in the test code itself.
+
+- How a file that is created by the test run can be captured together with other test results for later viewing.
+
+You can find the templates that generate these Java files in the <a href="https://github.com/galasa-dev/cli/tree/main/pkg/cmd/templates/projectCreate/parent-project/test-project/src/main/java" target="_blank">Galasa cli repository</a>.
+
+
+## Building the example projects by using Maven
+
+Complete the following steps to build the example projects:
+
+1. On the command line, navigate into the top level parent folder. In our example the command is ```cd dev.galasa.example.banking```
+1. Invoke Maven to build the OSGi bundles by running the ```mvn clean install``` command.
+
+The built artifacts are typically placed in the _~/.m2/repository_ folder.
+
+Note: The tool currently does not generate build files for use with Gradle. 
+
+
+## Additional notes on the key elements of pom.xml files
+
+The following sections provide a little more information about some of the elements that are found within the various pom.xml files.
+
+### The parent pom.xml file elements
+
+- The `<project>` and `<modelVersion>` elements are standard prologues to a `pom.xml` file.
+- The `<groupId>` is used to group related Maven projects in a Maven repository. It is recommended (but not enforced) that all projects in a test stream share the same `groupId`.
```

Comment:
We mention `test stream` here. Should we link to an explanation of that concept ?

Reply:
added link